rviz2 crashes when trying to visualize RobotModel with textures

Using ros2 eloquent from debs, I have tried both listening to a URDF published by robot_state_publisher on the /robot_description topic, and I've tried loaded the URDF directly from file for the Rviz RobotModel plugin.  In both cases, I get this error:
`
Parsing robot urdf xml string.
[ERROR] [rviz2]: ItemIdentityException: Cannot find a group named rviz_common in ResourceGroupManager::isResourceGroupInitialised at /tmp/binarydeb/ros-eloquent-rviz-ogre-vendor-7.0.3/obj-x86_64-linux-gnu/ogre-v1.12.1-prefix/src/ogre-v1.12.1/OgreMain/src/OgreResourceGroupManager.cpp (line 1770)
terminate called after throwing an instance of 'Ogre::ItemIdentityException'
  what():  ItemIdentityException: Cannot find a group named rviz_common in ResourceGroupManager::isResourceGroupInitialised at /tmp/binarydeb/ros-eloquent-rviz-ogre-vendor-7.0.3/obj-x86_64-linux-gnu/ogre-v1.12.1-prefix/src/ogre-v1.12.1/OgreMain/src/OgreResourceGroupManager.cpp (line 1770)
Aborted (core dumped)
`
I have tracked this down to Rviz bombing out anytime you have a material with a texture line like this:
```  
<material name="RollLinks">
    <texture filename="package://pr2_description/materials/textures/pr2_wheel_left.png"/>
</material>
```

--- a/rviz_default_plugins/src/rviz_default_plugins/robot/robot_link.cpp
+++ b/rviz_default_plugins/src/rviz_default_plugins/robot/robot_link.cpp
@@ -68,6 +68,8 @@
 #include "rviz_common/properties/vector_property.hpp"
 #include "rviz_common/interaction/selection_manager.hpp"
 
+#define RVIZ_RESOURCE_GROUP "rviz_rendering"
+
 using rviz_rendering::Axes;
 using rviz_rendering::Shape;
 
@@ -717,7 +719,7 @@ void RobotLink::loadMaterialFromTexture(
   Ogre::MaterialPtr & material_for_link, const urdf::VisualSharedPtr & visual) const
 {
   std::string filename = visual->material->texture_filename;
-  if (!Ogre::TextureManager::getSingleton().resourceExists(filename, "rviz_common")) {
+  if (!Ogre::TextureManager::getSingleton().resourceExists(filename, RVIZ_RESOURCE_GROUP)) {
     resource_retriever::Retriever retriever;
     resource_retriever::MemoryResource res;
     try {
@@ -739,7 +741,7 @@ void RobotLink::loadMaterialFromTexture(
       try {
         image.load(stream, extension);
         Ogre::TextureManager::getSingleton().loadImage(filename,
-          Ogre::ResourceGroupManager::DEFAULT_RESOURCE_GROUP_NAME,
+          RVIZ_RESOURCE_GROUP,
           image);
       } catch (Ogre::Exception & e) {
         RVIZ_COMMON_LOG_ERROR_STREAM("Could not load texture [" << filename << "]: " << e.what());

